strawberryfields/api/connection.py

```diff
@@ -16,22 +16,23 @@
 """
 from datetime import datetime
 import io
-import logging
 from typing import List
 
 import numpy as np
 import requests
+import logging
```

Comment:
As per the the Pylint warning (`standard import "import logging" should be placed before "import numpy as np"`):

```suggestion
import logging
import numpy as np
import requests
```